File: test/fixtures/comment-density/corpus/sample.py
#!/usr/bin/env python3
# Module comment

def greet(name: str) -> str:
    """
    Return a greeting.
    
    This is a docstring, not a comment technically,
    but many density checkers count it.
    """
    # build message
    return f"Hello, {name}"

URL = "http://not.a.comment"  # inline comment

File: test/fixtures/comment-density/corpus/sample.go
package main

import "fmt"

// Package-level comment
// spanning two lines

// greet returns a greeting
func greet(name string) string {
	// build the message
	return fmt.Sprintf("Hello, %s", name)
}

/* block comment in Go */
var url = "http://not.a.comment"
